### PR TITLE
BUG/TST/DOC: test, fix, and changelog mention for #1416

### PR DESCRIPTION
**Pull request by wasade**

### CODE REVIEW

File: skbio/tree/tests/test_tree.py [hunk 1]
```diff
@@ -52,6 +52,17 @@ def rotate_f(items):
         self.complex_tree = TreeNode.read(io.StringIO(
             "(((a,b)int1,(x,y,(w,z)int2,(c,d)int3)int4),(e,f)int5);"))
 
+    def test_bug_issue_1416(self):
```
Comment (ebolyen) [inline skbio/tree/tests/test_tree.py]: I really like this name.

**Comment by ebolyen** [comment]
@wasade you've angered the PEP8 gods. Otherwise this looks great! You can drop the `/DOC` part of your commit message, changelogs are assumed.

**Comment by mortonjt** [comment]
Just tested it on the example that I posted in #1416 .  I think this is good to go!

**Comment by wasade** [comment]
blah, thanks @ebolyen. Fixing. Will drop "DOC" tag too

Thanks @mortonjt for the quick verification!

**Comment by wasade** [comment]
...on the one hand, I copied/pasted the way to recreate the issue which is the ideal way to define the test. Clearly the PEP8 gods are selective in their punishment as they overlook the issues themselves...

File: skbio/tree/tests/test_tree.py [hunk 2]
```diff
@@ -52,6 +52,17 @@ def rotate_f(items):
...
+        tree = TreeNode.read([u'(((a,b,f,g),c),d);'])
```
Comment (ebolyen) [inline skbio/tree/tests/test_tree.py]: Drop this `u` as well. It's deprecated in Python 3.

**Comment by ebolyen** [comment]
The PEP8 gods are a petty bunch.

**Comment by ebolyen** [comment]
:+1:

**Comment by ebolyen** [comment]
Thanks @wasade!

**Comment by mortonjt** [comment]
Thanks!

**Pull request merged**